tasks/tests.yml

```diff
@@ -9,7 +9,7 @@
 - name: "Ensure haproxy.cfg contains 'ansible managed' line"
   lineinfile:
     path: /etc/haproxy/haproxy.cfg
-    line: "# Ansible managed: Do NOT edit this file manually!"
+    line: "# {{ ansible_managed }}"
```

Comment:
I don't think we can check for existence of this line since `ansible_managed` is accessible only in `template` module and not in `ansible_facts`.